contents/blog/group-analytics-launch.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2021-12-15
+title: Group analytics are now available in PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["joe-martin"]
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+---
+
+Today, we’re excited to announce that PostHog now supports group analytics for both self-hosted users and those on PostHog Cloud. 
+
+Group analytics are used for tracking and analyzing multiples of any event type within PostHog. Unlike cohorts, which essentially create lists of users, groups can track a far wider range of information because they apply to _events_, rather than users. 
```

Comment:
> they apply to _events_, rather than users.  

I'm not sure I get this phrasing?

Reply:
```suggestion
Group analytics are used for tracking and analyzing multiples of any event type within PostHog. Unlike cohorts, which essentially create lists of users, groups can track a far wider range of information because they apply at the event level. 
```

Does that read a little clearer @jamesefhawkins?

I'm pulling a little from the docs draft - https://github.com/PostHog/posthog.com/pull/2528/files - but the intent is to make it clear that groups track events because that enables some more interesting analysis opportunities than purely company-based groups.